### PR TITLE
feature(concept-cat): Added support for field SKOSNO:Omfang (range)

### PR DESCRIPTION
**Pull request by thomasnokleby**

### CODE REVIEW

File: applications/concept-cat/src/main/java/no/ccat/service/RDFToModelTransformer.java [hunk 1]
```diff
@@ -263,12 +280,12 @@ private ContactPoint extractContactPoint(Resource resource) {
 
             Statement phoneStatement = contactPointResource.getProperty(VCARD4.hasTelephone);
             if (phoneStatement != null) {
-                String parsedPhoneNumber = parseURIFromStatement(phoneStatement);
+                String parsedPhoneNumber = getOnlySchemeSpesificPartOfURIFromStatement(phoneStatement);
```
Comment (stigbd) [inline applications/concept-cat/src/main/java/no/ccat/service/RDFToModelTransformer.java:283]: Does this change belong to the omfang-feature? If not, maybe move to another PR?

**Review by frodegill — APPROVED** [review]

File: applications/concept-cat/src/main/java/no/ccat/service/RDFToModelTransformer.java (hunk 1 above)
Comment (thomasnokleby) [reply applications/concept-cat/src/main/java/no/ccat/service/RDFToModelTransformer.java:283]: Yes. After this change, two different parts of the code use the URI parse function, in subltly different ways. So I split it in 3, one base part, and two apropriately named methods that use the base.

**Review by azaxarov — APPROVED** [review]
